packages/blade/src/components/ChatBubble/SelfMessageBubble.web.tsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import BaseBox from '~components/Box/BaseBox';
+import { AlertCircleIcon } from '~components/Icons';
+import { Text } from '~components/Typography';
+
+const SelfMessageBubble = ({
+  children,
+  isError,
+  errorText = 'Message not sent. Tap to retry.',
+  onClick,
+  messageType,
+}: {
+  children: React.ReactNode | string;
+  isError?: boolean;
+  errorText?: string;
+  onClick?: () => void;
+  messageType: 'last' | 'default';
+}): React.ReactElement => {
+  return (
+    <BaseBox
+      display="flex"
+      gap="10px"
+      flexDirection="column"
+      onClick={onClick}
```

Comment:
when onClick is present we should change the chat bubble to become a button for a11y reasons.

Reply:
done.